pennylane/decomposition/__init__.py

```diff
@@ -21,8 +21,33 @@
 
     This module is experimental and is subject to change in the future.
 
-.. TODO::
-    Add section here explaining how to enable and disable the new decomposition system. [sc-83993]
+To activate and deactivate the new experimental graph-based decomposition system, use the switches
```

Comment:
@isaacdevlugt This completes the missing sections in the module doc-string, can you take a look when you have time.